server/app.js

```diff
@@ -12,39 +13,52 @@ app.use(bodyParser.urlencoded({ extended: true }));
 app.use(morgan('tiny'));
 app.use(express.static('public'));
 
-app.get('/api/movie', (req, res) => {
-  const { id } = req.query;
-  db.getCasts(id)
-    .then(data => {
-      if (data.length === 0) {
-        res.sendStatus(404);
-      } else {
-        data[0].casts.sort((a, b) => a.role - b.role);
-        res.send(data[0].casts);
-      }
-    })
-    .catch(err => {
-      console.error('>>>', err);
-      res.sendStatus(500);
-    });
-});
+// app.get('/api/movie', (req, res) => {
+//   const { id } = req.query;
+//   db.getCasts(id)
+//     .then(data => {
+//       if (data.length === 0) {
+//         res.sendStatus(404);
+//       } else {
+//         data[0].casts.sort((a, b) => a.role - b.role);
+//         res.send(data[0].casts);
+//       }
+//     })
+//     .catch(err => {
+//       console.error('>>>', err);
+//       res.sendStatus(500);
+//     });
+// });
 
-app.post('/api/movie', (req, res) => {
-  console.log(req.body);
-  db.saveMovie(req.body);
-  res.sendStatus(200);
-});
+// app.post('/api/movie', (req, res) => {
+//   console.log(req.body);
+//   db.saveMovie(req.body);
+//   res.sendStatus(200);
+// });
 
-app.put('/api/movie', (req, res) => {
-  console.log(req.query);
-  db.modifyMovie(req.query);
-  res.sendStatus(200);
-});
+// app.put('/api/movie', (req, res) => {
+//   console.log(req.query);
+//   db.modifyMovie(req.query);
+//   res.sendStatus(200);
+// });
 
-app.delete('/api/movie', (req, res) => {
-  console.log(req.query);
-  db.deleteMovie(req.query.id);
-  res.sendStatus(200);
+// app.delete('/api/movie', (req, res) => {
+//   console.log(req.query);
+//   db.deleteMovie(req.query.id);
+//   res.sendStatus(200);
+// });
+
+/* RESTFUL API FOR POSTGRESQL */
+
+app.get('/api/movie', (req, res, next) => {
+  const { id } = req.query;
+  pool.query(`SELECT * FROM movies WHERE movie_id = ${id};`, (err, results) => {
+    if (err) {
+      res.sendStatus(404);
+    } else {
+      res.status(200).send(results);
+    }
+  });
 });
```

Comment:
Why is this query in your server file? Wouldn't it be better in a database file? @DavidFuent1

Reply:
@RachNovo I plan on moving it over for modularity, was just trying to get a path in for testing.